### DISCUSSION TITLE
Map of the Day Recommendation

### DISCUSSION THREAD

**Discussion by Rekunan** [ppy/osu#22615] [Ideas]
If the player is online and then selects singleplayer, it'd recommend a random map that has a high rating (probably 7.0+) and is within the `Recommended difficulty` range. By default, it'd be limited to featured artists only, with a setting to allow non-featured artist maps.

This will help maps/mappers get more recognition from the less informed players, and is another reward for using featured artists.

**Comment by monochrome22** [comment]
I don't see how recommending a random map is any different from simply hitting F2. 

In my opinion, this idea only makes sense if someone with a high level of mapping experience manually picks a map that they think deserves more recognition. 

As for the difficulty range, there would be 2 solutions. Maps could be picked for each difficulty range, Meaning, map of the day for 2-4 star, map of the day for 4-6 star, map of the day for 6-8 star. Other solution would be to pick the map from a wide difficulty range, like 2-8 stars. This would mean every few days the map should be within range of even lower skilled players. In my opinion, the second solution is better.

**Reply by bdach** [reply]
I'm guessing this would be a recommendation of an *online* map. As in one you don't have downloaded?

**Reply by monochrome22** [reply]
That would make slightly more sense, but it would still be pretty pointless. Again, it wouldn't be any different from hitting F2 and finding a map that you haven't played before. It would only be good if the Map of the Day was manually picked.

**Reply by Rekunan** [reply]
The other difference is that it'd only recommend maps that have been highly rated, using the user rating system.

**Reply by monochrome22** [reply]
Most maps have a high user rating. You'd need to go through multiple maps to find something below 9.0. 

I just think there should be something special about "Map of the Day". If it was a random map picked based on filters, then it wouldn't be special, it would be just another map. It would only be special if it was handpicked by someone with a lot of experience who thought it's a fun, well made, lesser-known map that you should try.

**Reply by Rekunan** [reply]
I think yeah this is the better route, thanks for the good points. We could even use existing groups such as project loved captains.

**Reply by monochrome22** [reply]
One idea would be to give a group of people access to a page where they can submit maps. So every day, the map of the day system would pick the next map in the list. The group of people could be beatmap nominators, project loved captains, or maybe just the entire staff. 
It would be too much work for any group to have to decide on a map every single day, that's why a page would be good. If someone from the group comes across an exceptional map, then they can go to the page and submit it.

**Comment by peppy** [comment]
I do like this concept.
